aman/src/components/Input.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Input({ type, name, value, onChange, placeholder, className }) {
+  return (
+    <div>
+      <input
+        type={type}
+        name={name}
+        value={value}
+        onChange={onChange}
+        className={`border-gray-300 border-[1px] flex h-10 w-full rounded-md px-3 py-2 text-sm ${className}`}
```

Comment:
Here, in border-[1px], as informed earlier try to use the dimensions, colors etc available in tailwind css. If the exact dimension is not available, then tyr to use the nearest value.